app/views/content_items/coming_soon.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= content_for :page_class, "coming-soon" %>
+<%= content_for :title, "Coming soon - GOV.UK" %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/title', title: 'Coming soon' %>
+
+    <p class="summary">
+      This document will be published on<br />
+      <%= content_tag(:abbr, class: 'publish_time', title: @content_item.iso8601_publish_time) do %>
```

Comment:
Nice. Does it make sense to duplicate the iso8601 datetime in a `title` attribute so that it still gets displayed on hover? Or is that something nobody is really interested in?